content/en/admin/config.md

```diff
@@ -387,7 +387,7 @@ See the [ElasticSearch setup page for details on each setting](../elasticsearch#
 
 #### `ES_HOST`
 
-Host of the Elasticsearch server. Defaults to `localhost`
+Host of the ElasticSearch server. Defaults to `localhost`. If using TLS, prepend the hostname with `https://`. For example: `https://myelastic.tld`.
```

Comment:
Elasticsearch is proper usage. Also lets use example.com domain as proposed in #1227 for consistency.

```suggestion
Host of the Elasticsearch server. Defaults to `localhost`. If using TLS, prepend the hostname with `https://`. For example: `https://elastic.example.com`.
```